internal/local/activate.go

```diff
@@ -136,6 +136,10 @@ func (service *ProvisioningService) GetGeneralSettings() (general.Response, erro
 	if err != nil {
 		return general.Response{}, err
 	}
+	if service.flags.Verbose {
```

Comment:
You shouldn't need the condition -- the log level sets when these should be logged